src/docker/Dockerfile

```diff
@@ -96,6 +96,11 @@ RUN curl -sL https://packages.microsoft.com/keys/microsoft.asc | \
     tee -a /etc/apt/sources.list.d/azure-cli.list && \
     install_packages azure-cli=${AZURE_CLI}
 
+# Install Nodejs and NPM
+ARG NODEJS_VERSION=10.15.2~dfsg-2
```

Comment:
this is not the LTS

Reply:
It is the latest version in the debian buster package library, but you make a good point. I will look into installing from another source, perhaps a different PPA

Reply:
Actually it may be a better idea to install [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md)

Reply:
AFAIK and why I did not this yet is that node uses multipile PPA for each of their releases streams. Current, LTS and still maintained maintance streams.

NVM seems to be a valid approach.